plugins/ruby-foreman-host-reports/debian/rules

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/make -f
+# -*- makefile -*-
+# Sample debian/rules that uses debhelper.
+# This file was originally written by Joey Hess and Craig Small.
+# As a special exception, when this file is copied by dh-make into a
+# dh-make output file, you may use that output file without restriction.
+# This special exception was added by Craig Small in version 0.37 of dh-make.
```

Comment:
I think we should drop these from our rules files. These are nothing like the original anymore.

Reply:
Yeah sure, dropped.